src/common/com/intellij/plugins/haxe/lang/psi/HaxeResolver.java

```diff
@@ -250,7 +250,13 @@ private PsiFile resolveImportFile(HaxeReference reference) {
         VirtualFile importVFile = importDir == null ? null : importDir.findChild(fileName);
         importPsiFile = importVFile == null ? null : PsiManager.getInstance(reference.getProject()).findFile(importVFile);
         if (importPsiFile != null) {
-          break;
+          // for addition case-sensetive check because find file is not case-sensetive
+          if(!fileName.equals(importPsiFile.getName())) {
```

Comment:
OK.  I tested it.  It works much better than it used to.  I guess we can close the bugs, but we *should* have one for getting the resolution order correct.